### PR TITLE
chore: Inline SetupAlpineHostPath

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

It does not make sense for SetupAlpineHostPath to be in the global testsuite cleanup, therefore remove it from there and inline it in the only other place where is is used.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by 0xFelix** [comment]
This is a test, let's see if removing `SetupAlpineHostPath` from the global testsuite cleanup breaks anything.

/cc @fossedihelm

**Comment by fossedihelm** [comment]
wow, looks great!

File: tests/utils.go [hunk 1]
```diff
@@ -103,15 +103,8 @@ const (
 	DiskAlpineHostPath     = "disk-alpine-host-path"
 	DiskWindowsSysprep     = "disk-windows-sysprep"
 	DiskCustomHostPath     = "disk-custom-host-path"
```
Comment (dankenigsberg) [inline tests/utils.go:105]: Thanks! I hope that one day, these public constants would follow.

**Comment by codingben** [comment]
Hi, nice catch. Why your PR title is `chore: Inline SetupAlpineHostPath` and not `test: inline SetupAlpineHostPath` or `tests: inline SetupAlpineHostPath`?

**Comment by 0xFelix** [comment]
Was trying to use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), though it is debatable what to use. But we don't have a clear policy for this either.

File: tests/storage/storage.go [hunk 2]
```diff
@@ -82,10 +82,10 @@ const (
 	// Without cloud init user data Cirros takes long time to boot,
 	// so provide this dummy data to make it boot faster
 	cirrosUserData = "#!/bin/bash\necho 'hello'\n"
-)
 
-const (
-	diskSerial = "FB-fb_18030C10002032"
+	defaultDiskSize  = "1Gi"
+	diskSerial       = "FB-fb_18030C10002032"
+	osAlpineHostPath = "alpine-host-path"
```
Comment (alicefr) [inline tests/storage/storage.go:88]: Only commet I have is if it is necessary to declare the constant or we can simply hardcode the values in the function call. 
I don't have an opinion, it only removes 2 lines and avoid to scroll to look for the values

**Comment by alicefr** [comment]
/approve
Just put on hold if you want to modify it, but feel free to unhold if you prefer this version. Thanks!
/hold

File: tests/storage/storage.go (hunk 2 above)
Comment (0xFelix) [reply tests/storage/storage.go]: I changed the consts as suggested. I like it better this way too.

**Comment by alicefr** [comment]
Thanks!
/unhold
/lgtm

**Comment by fossedihelm** [comment]
@0xFelix can you please try to rebase? Thanks

**Comment by 0xFelix** [comment]
@fossedihelm Done

**Comment by brianmcarey** [comment]
/hold

These compute lanes have failed on the same test for a number of runs in a row - could you take a look @0xFelix ?

**Comment by akalenyu** [comment]
> I don't see how the changes in this PR are related to failures in the VM rollout strategy tests at all. These tests use Cirros and change the amount of sockets to cause a restart required condition on the VM?

There was this PR to try handle it:
https://github.com/kubevirt/kubevirt/pull/11171
/cc @fossedihelm

**Comment by brianmcarey** [comment]
> > I don't see how the changes in this PR are related to failures in the VM rollout strategy tests at all. These tests use Cirros and change the amount of sockets to cause a restart required condition on the VM?
> 
> There was this PR to try handle it: #11171 /cc @fossedihelm

Ok sorry - I do believe theres a PR waiting in kubevirtci - https://github.com/kubevirt/kubevirtci/pull/1129 - to address this flake. Not sure why the flake was exposed so much in this PR though. 

/retest

**Comment by 0xFelix** [comment]
I added the commit of @fossedihelm in #11171 to this PR. Let's see if it helps. Indeed, interesting why the flake was exposed so much in this test.

**Comment by fossedihelm** [comment]
Thanks! Let's see the 1.29 compute result and then, I will lgtm, just to not overload the CI, if the fix does not work!

**Comment by fossedihelm** [comment]
/lgtm
The compute failure is another flake...
Let's see other k8s providers

File: tests/hotplug/rolloutstrategy.go [hunk 3]
```diff
@@ -63,6 +65,7 @@ var _ = Describe("[Serial][sig-compute]VM Rollout Strategy", decorators.SigCompu
 			vm := libvmi.NewVirtualMachine(vmi, libvmi.WithRunning())
 			vm, err := virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm)
 			Expect(err).NotTo(HaveOccurred())
+			libwait.WaitUntilVMIReady(vmi, console.LoginToCirros)
```
Comment (fossedihelm) [inline tests/hotplug/rolloutstrategy.go:68]: My bad!
different failure:
```suggestion
			Eventually(func() error {
				vmi, err = kubevirt.Client().VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, &metav1.GetOptions{})
				return err
			}, 120*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
			libwait.WaitUntilVMIReady(vmi, console.LoginToCirros)
```

**Pull request merged**